prawns/identities/src/handlers/identity.clear.identity.extrinsic.ts

```diff
@@ -1,30 +1,41 @@
-import { ExtrinsicHandlerContext } from '@subsquid/substrate-processor';
+import { CallHandlerContext } from '@subsquid/substrate-processor';
 import { decodeAddress } from '../utils';
-import { SubstrateIdentity, SubstrateNetwork, SubstrateIdentityAction } from '../model';
-import { getManager } from 'typeorm';
+import {
+  SubstrateIdentity,
+  SubstrateNetwork,
+  SubstrateIdentityAction,
+} from '../model';
+import { Store } from '@subsquid/typeorm-store';
+import substrateIdentityRepository from '../repositories/substrateIdentityRepository';
+import getCallOriginAccount from '../utils/getCallOriginAccount';
+import assert from 'assert';
 
 export default (network: SubstrateNetwork) =>
-  async (ctx: ExtrinsicHandlerContext) => {
-
+  async (ctx: CallHandlerContext<Store>) => {
     const blockNumber = BigInt(ctx.block.height);
     const date = new Date(ctx.block.timestamp);
-    const account = ctx.extrinsic.signer;
-    const rootAccount = decodeAddress(account);
+    const account = getCallOriginAccount(ctx.call.origin, network);
+    assert(account);
```

Comment:
I guess we need the signer account to be there. Should we stop the whole indexing when it's not? I guess that was what was happening on the previous indexing, because `account` could not be null/undefined.

Another alternative would be to skip that call, just by doing `return` from this. Is that an option in your opinion?